src/Compilers/Core/Portable/Symbols/WellKnownMemberNames.cs

```diff
@@ -198,6 +198,11 @@ public static class WellKnownMemberNames
         /// </summary>
         public const string OnesComplementOperatorName = "op_OnesComplement";
 
+        /// <summary>
+        /// The name assigned to the Modulus operator.
```

Comment:
Copy paste error?

Reply:
👍 - I quick slapped a fix of this onto the next PR https://github.com/dotnet/roslyn/pull/23472 https://github.com/dotnet/roslyn/pull/23472/commits/7c8bd8027c888605ac9a0c132b3c3d381a164a56